src/main/java/cam72cam/immersiverailroading/entity/EntityRollingStock.java

```diff
@@ -78,6 +125,14 @@ public void onUpdate() {
 			}
 		}
 	}
+	
+	public boolean playerHasOp (EntityPlayer player) {
+		return  FMLCommonHandler.instance().getMinecraftServerInstance().getPlayerList().getOppedPlayers().getEntry(player.getGameProfile()) != null;
```

Comment:
Is this safe on both the client and server side?

Reply:
it works in single player but I've never tested it on an actual server